lib/src/main/java/com/diffplug/spotless/java/GoogleJavaFormatStep.java

```diff
@@ -109,17 +119,17 @@ public static boolean defaultReflowLongStrings() {
 		final String style;
 		final boolean reflowLongStrings;
 
-		State(String stepName, String version, Provisioner provisioner) throws Exception {
-			this(stepName, version, DEFAULT_STYLE, provisioner);
+		State(String stepName, String artifact, String version, Provisioner provisioner) throws Exception {
+			this(stepName, artifact, version, DEFAULT_STYLE, provisioner);
 		}
 
-		State(String stepName, String version, String style, Provisioner provisioner) throws Exception {
-			this(stepName, version, style, provisioner, DEFAULT_REFLOW_LONG_STRINGS);
+		State(String stepName, String artifact, String version, String style, Provisioner provisioner) throws Exception {
+			this(stepName, artifact, version, style, provisioner, DEFAULT_REFLOW_LONG_STRINGS);
```

Comment:
The point of all these delegate constructors is backward compatibility, none of them should be modified. Instead, there should be only two changes:

1) the new constructor which adds `String mavenCoordinate`
2) the previous constructor should pass `MAVEN_COORDINATE` as the default value for that